openmetadata-docs/content/v1.5.x/menu.md

```diff
@@ -919,6 +919,8 @@ site_menu:
     url: /releases/supported
   - category: Releases / All Releases
     url: /releases/all-releases
+  - category: Releases / All Releases / 1.5.2 Release
```

Comment:
Just to confirm here it is` 1.5.2 `only, right?

Reply:
Yes, v.1.5.3 is tagged as latest. So we add the older release in the menu index